repo2docker/buildpacks/base.py

```diff
@@ -143,6 +144,15 @@ class BuildPack(LoggingConfigurable):
         """
     )
 
+    base_image = Unicode(
+        "buildpack-deps:zesty",
```

Comment:
as the topic of the original PR was to bump the base image to artful, shouldn't this be artful?

Reply:
See https://github.com/jupyter/repo2docker/pull/148#issuecomment-356143965 :)

Reply:
oh I see what you mean